cmd/check_spec.go

```diff
@@ -174,6 +173,29 @@ func runCheckSpecCmd(_ *cobra.Command, _ []string) error {
 
 	// TODO: more checks
 
+	workflows := Config.Construction.Workflows
+	keypair, _ := keys.GenerateKeypair("secp256k1")
```

Comment:
hardcoded it for now, need injection from somewhere

Reply:
I remember .ros file has curve_type in its workflow def, can we use it?